### PR TITLE
[Duplicate]Getting started with Python plug-ins for GIMP

### PR DESCRIPTION
**Pull request by kmwangi-ken**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #5477`

### CODE REVIEW

**Comment by kmwangi-ken** [comment]
@mercymeave,

I have committed all the requested changes. I request that you can have a look at it.

**Comment by kmwangi-ken** [comment]
@mercymeave,
Hi.. I was checking on the update. It was a duplicate on pr #5477 . Keep me posted. Thank you

cc
@hectorkambow

File: content/articles/getting-started-with-python-plug-ins-for-gimp/index.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+"<Image>/Image/Resize to max ", 
+# The types of images the plug-in is made to handle 
+"RGB*, GRAY*",
+# The parameters for the plug-in’s method
+[], 
+# The results of the plug-in’s method
+[], 
+# The name of the method to call in your Python code
+gimp_plugins) 
+main()
```
Comment (mercymeave) [inline content/articles/getting-started-with-python-plug-ins-for-gimp/index.md:60]: This piece of code won't run. Use the correct python indentations

File: content/articles/getting-started-with-python-plug-ins-for-gimp/index.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
+### Introduction 
+GIMP is an open-source image manipulation tool that many people use as a viable alternative to some of the commercial offerings. GIMP is efficient in dealing with tangled elements such as layers and paths. It also supports several different image formats and comes with relatively complex filters.
```
Comment (mercymeave) [inline content/articles/getting-started-with-python-plug-ins-for-gimp/index.md:2]: Provide the alternatives. like pillow etc.

Comment (mercymeave) [inline content/articles/getting-started-with-python-plug-ins-for-gimp/index.md:2]: What exactly do you mean by the term `commercial offerings`?

File: content/articles/getting-started-with-python-plug-ins-for-gimp/index.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+![Plug-ins folder](/section-engineering/getting-started-with-python-plug-ins-for-gimp/plugins.png)
+
+Duplicate the highlighted path and paste it on your file explorer pathway. Make another text document in the open folder and save it as a `.py`  On Mac and Linux systems, that folder is at `~/.gimp-2.6/folder`.
+
+There are 12 arguments required in your work register. You can read more about it [here] (http://www.expmedia.com/content/extending-gimp-python-python-fu-modules-region-2). You must use your Python script in conjunction with GIMP to add the module to one of the GIMP menus. Following that is a model.
+
+```python
+#!/usr/bin/python
+From gimpfu import * # same for every module
```
Comment (mercymeave) [inline content/articles/getting-started-with-python-plug-ins-for-gimp/index.md:35]: This is not how we import libraries in python.

**Review by mercymeave — CHANGES_REQUESTED** [review]
Hey @kmwangi-ken 

Please of go over all the changes request here and in subsequent PRs for this article you seem to have missed all the changes requested by @srishilesh  in the very first PR and my requests as well. The screenshots below show the changes requested in the previous PRS but no action was taken by the author 
Screenshot from 2021-12-26 15-35-41

Screenshot from 2021-12-26 15-40-13
Being a duplicate PR with errors, we will close it until all the corrections are made in your forked repo.

CC: @hectorkambow

**Pull request closed without merging**